### DISCUSSION TITLE
How to perform a multi-threaded calculation?

### DISCUSSION THREAD

**Discussion by qiuqiubing** [Anura3D/Anura3D_OpenSource#42] [Q&A] [answered]
Dear all,
Can the program perform multi-threaded calculations? How does it work? Thank you.

**Comment by murphy-john** [comment, marked as answer]
Hello @qiuqiubing! Unfortunately, Anura3D cannot be run on multi-treads. Adding this feature is high on our priority list of features. We are hopeful we will be adding this capability soon.

**Comment by qiuqiubing** [comment]
Thank you! And we are looking forward to it~


Best,
qiuqiu



---Original---
From: "John ***@***.***&gt;
Date: Fri, Nov 4, 2022 23:52 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Anura3D/Anura3D_OpenSource] How to perform a multi-threaded calculation? (Discussion #42)




 
Hello @qiuqiubing! Unfortunately, Anura3D cannot be run on multi-treads. Adding this feature is high on our priority list of features. We are hopeful we will be adding this capability soon.
